File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        boolean isTrue = false;
        for(int i=0; i<nums.length-1; i++){
            for(int j=i+1; j<nums.length; j++){
                if(nums[i] == nums[j]){
                    isTrue =  true;
                    break;
                }
                   
            }
        }
        return isTrue;
    }
}

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        map = {}
        res, count = 0, 0
        n = len(nums)
        for num in nums:
            map[num] = map.get(num,0) + 1
            res = num if map[num] > count else res
            count  = max(map[num], count)
        return res
        
        